Atomic/Export.py

```diff
@@ -25,6 +25,17 @@ def export_docker(graph, export_location):
         os.makedirs(export_location)
 
     try:
+        dangling_images = DOCKER_CLIENT.images(filters={"dangling":True}, quiet=True)
+        if dangling_images:
+            util.writeOut("There are dangling images in your system. Would you like atomic to prune them [y/N]")
+            choice = sys.stdin.read(1)
+            if choice.lower() == 'y':
+                for key in dangling_images:
+                    util.writeOut("Deleting dangling image with ID: {0}".format(key))
+                    subprocess.check_call(["/usr/bin/docker", "rmi", "-f", key])
```

Comment:
On 01/14/2016 10:20 AM, Shishir wrote:
>
> In Atomic/Export.py
> <https://github.com/projectatomic/atomic/pull/284#discussion_r49738363>:
>
> > @@ -25,6 +25,17 @@ def export_docker(graph, export_location):
> >          os.makedirs(export_location)
> >  
> >      try:
> > +        dangling_images = DOCKER_CLIENT.images(filters={"dangling":True}, quiet=True)
> > +        if dangling_images:
> > +            util.writeOut("There are dangling images in your system. Would you like atomic to prune them [y/N]")
> > +            choice = sys.stdin.read(1)
> > +            if choice.lower() == 'y':
> > +                for key in dangling_images:
> > +                    util.writeOut("Deleting dangling image with ID: {0}".format(key))
> > +                    subprocess.check_call(["/usr/bin/docker", "rmi", "-f", key])
>
> @rhatdan <https://github.com/rhatdan> can we remove |/usr/bin| ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projectatomic/atomic/pull/284/files#r49738363>.
>
Yes, we have another patch that went in to remove all /usr/bin/docker